lib/view/staff_welfare/your_welfare_view.dart

```diff
@@ -23,11 +24,21 @@ class YourWelfareView extends StatelessWidget {
         iconTheme: Styles.appBarIconTheme,
         title: Text(
           _title,
-          style: Styles.appBarTextStyle,
+          style: Styles.textSemiBold.copyWith(color: AppColors.blackAlpha800),
         ),
         actions: <Widget>[
           IconButton(
-            icon: Icon(Icons.call),
+            icon: Stack(
```

Comment:
Would it be worth making this stack into a widget? I think we do something similar with PDF icon on another screen?

Reply:
No, this is definitely specific to this icon. I'm combining an arrow with a phone to make it look like the mock, because I can't extract the image asset from zeplin